src/System/RequirementsManager.php

```diff
@@ -160,6 +161,12 @@ function ($directory) {
             true,
             __('Slightly enhance performances.')
         );
+        $requirements[] = new ExtensionConstant(
```

Comment:
Should be moved before the `// Below requirements are optionals` comment (probably after the `zlib` extension check).